packages/fluentui/react-northstar/src/components/Portal/usePortalBox.ts

```diff
@@ -25,15 +25,24 @@ export const usePortalBox = (options: UsePortalBoxOptions): HTMLDivElement | nul
   }, [target]);
 
   useIsomorphicLayoutEffect(() => {
+    const classes = className.split(' ').filter(Boolean);
+
     if (element) {
-      element.setAttribute('class', className);
+      element.classList.add(...classes);
```

Comment:
Had to rework this section because otherwise `.setAttribute()` overwrites all classes and caused glitches during theme switches.